### PR TITLE
(Fix) Update README.md

### PR DESCRIPTION
**Pull request by vbaranov**
- Remove screenshots of dapp
- Refine description of wallets section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,16 @@
 
 * Google Chrome v 59.0.3071.115+
 
-## MetaMask plugin setup
+## MetaMask/Nifty Wallet extensions setup
 
-* Connect to POA Network in MetaMask (See [Connect to POA Network via MetaMask](https://github.com/poanetwork/wiki/wiki/POA-Network-on-MetaMask)) or Nifty Wallet extensions (See [Connect to POA Network via Nifty Wallet](https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet))
+* Connect to POA Network in MetaMask extension (See [POA Network on MetaMask](https://github.com/poanetwork/wiki/wiki/POA-Network-on-MetaMask)) or Nifty Wallet extension (See [POA Network on Nifty Wallet](https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet))
 
 * Import your initial key to MetaMask or to Nifty Wallet extensions: browse keystore file, received from invitation, and enter password for the key (See Importing of keys section [in MetaMask](https://github.com/poanetwork/wiki/wiki/POA-Network-on-MetaMask#importing-of-keys) or [in Nifty Wallet](https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet#importing-of-keys)).
```
Comment (varasev) [inline README.md]: The link for `Nifty Wallet` here should point to https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet#importing-keys

**Comment by varasev** [comment]
The link to `./src/addresses.js` is broken: https://github.com/poanetwork/poa-dapps-keys-generation/blob/269332a4766ffdee1e7294b151751c09831dc9b3/README.md#configuration-file

File: README.md (hunk 1 above)
Comment (vbaranov) [reply README.md]: Thanks, fixed the link

**Comment by varasev** [comment]
@vbaranov There is also broken link to `./src/addresses.js`: https://github.com/poanetwork/poa-dapps-keys-generation/blob/269332a4766ffdee1e7294b151751c09831dc9b3/README.md#configuration-file

We need to change its title to `./src/utils/addresses.js` and change its href to `https://github.com/poanetwork/poa-dapps-keys-generation/blob/e7577427fd28c6e9c52c2b10572cda6128b21ede/src/utils/addresses.js`.

Maybe the whole section `Configuration file` should be removed from README? I don't quite understand what this section for.

**Comment by vbaranov** [comment]
@varasev 

This section has been added for those who want to build their own POA based Network with the trusted ceremony setup. But I agree that it is redundant here because we have a guide [how to bootstrap POA based Network in our wiki](https://github.com/poanetwork/wiki/wiki/Master-of-Ceremony-Setup), outdated though.

I removed "Configuration file" section from README https://github.com/poanetwork/poa-dapps-keys-generation/pull/89/commits/d710b2b968eaf18f09ef350d10f9b3b6313e1b85.

**Review by varasev — APPROVED** [review]

**Pull request merged**